app/steps/ui/deceased/dob/template.html

```diff
@@ -4,7 +4,7 @@
 
 {% block question %}
     <h1 class="form-title heading-large">{{content.question}}</h1>
-    <p class="form-hint">{{content.questionHint}}</p>
+    <p class="form-hint no-link">{{content.questionHint}}</p>
```

Comment:
Can we add the form hint into the `date` mixin?

Reply:
Hi @shepda 
They've been moved into the date mixin now.